### DISCUSSION TITLE
create workspace

### DISCUSSION THREAD

**Discussion by Foni86** [Godiesc/firefox-one#45] [Q&A]
I may simply be an idiot, but in your screenshots I see that you have work spaces like those in opera, "tableros" in Spanish. But I don't see how the hell to activate or use them

**Comment by Godiesc** [comment]
hi,
Firefox don't have workspaces, one of the icons in my ScreenShots is of [Simple Tab Groups](https://addons.mozilla.org/es/firefox/addon/simple-tab-groups/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search) extension and the other is just the bookmarks menu icon.
